### PR TITLE
Bootstrap panel

### LINKED ISSUE #673: Bootstrap Panel UI for steps
Following up on a lot of changes from the past week; I am thinking of how we could integrate a lot of the buttons and features into a better and more coherent design using Bootstrap Panels:

https://getbootstrap.com/docs/3.3/components/#panels-heading

Maybe like this?

image

I /think/ this would work:

```html
<div class="panel panel-default">
    <div class="panel-heading">
        <h3 class="panel-title">Module name <span class="caret"></span></h3>
        <a class="pull-right btn btn-xs btn-default"><span class="fa fa-trash" aria-hidden="true"></span></a>
    </div>

    <div class="panel-body"></div>

    <div class="panel-footer">
        <a class="btn btn-sm btn-default">Save</a><a class="pull-right btn btn-sm btn-default">
        <span class="fa fa-plus" aria-hidden="true"></span> Insert module</a>
    </div>
</div>
```

This may also give us more flexibility to add more tools and such as the project gets more complex.

### PR DESCRIPTION
**Pull request by Harshithpabbati**
Fixes #673 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by Harshithpabbati** [comment]
@jywarren @publiclab/is-reviewers see this. I just made new pr because of the dist files conflict.

**Comment by Harshithpabbati** [comment]
@jywarren ready to merge.

**Comment by Harshithpabbati** [comment]
@harshkhandeparkar approve this.

File: examples/demo.css [hunk 1]
```diff
@@ -1,258 +1,272 @@
...
+  font-weight: bold;
+  font-style: normal;
 }
 
 body {
-  padding: 20px;
-  margin: 0 auto 200px;
-  max-width: 1000px;
-  background: #f8f8fa;
+padding: 20px;
```
Comment (HarshKhandeparkar) [inline examples/demo.css]: Why did you remove the indentation?

Comment (Harshithpabbati) [reply examples/demo.css]: That was by mistake.

Comment (HarshKhandeparkar) [reply examples/demo.css]: Can you undo that?

**Comment by Harshithpabbati** [comment]
@jywarren see this.

**Comment by jywarren** [comment]
Hi! Can you post a screenshot of the current state? Sorry to ask, but just
want to be sure after all the rebasing. THANK YOU for your hard work!!!

**Comment by Harshithpabbati** [comment]
pf
Final output :smile: .

**Comment by Harshithpabbati** [comment]
@jywarren see this.

**Comment by Harshithpabbati** [comment]
@jywarren Please look at this.

**Comment by Harshithpabbati** [comment]
@jywarren merge this

**Comment by jywarren** [comment]
Hi @harshithpabbati I'm sorry it's been very busy with visiting coworkers here, I'm not always available to do merges. Let me take a look now!

**Comment by jywarren** [comment]
OK, I've pushed up a few small tweaks to the CSS which I think makes things a bit tidier:

image

**Comment by jywarren** [comment]
This is now ready to merge once it passes tests!

**Comment by jywarren** [comment]
🎉 🎉 This is awesome!!!! Thanks @Harshithpabbati and also @HarshKhandeparkar and others who helped make this happen!!!

**Comment by jywarren** [comment]
@publiclab/is-reviewers I just merged to stable, built dist files, published to NPM, and merged to gh-pages and published the demo. It's a longer process now so I hope i got it all right!
